doc/newsfragments/2003_changed_zk_env.rst

```diff
@@ -0,0 +1 @@
+* Pass driver's custom environment to :py:class:`ZookeeperStandalone <testplan.testing.multitest.driver.zookeeper.ZookeeperStandalone>` stopping function.
```

Comment:
Thanks for the PR! Can you drop the asterisk from the news fragment and rename the 
file to 2003_changed.zk_env.rst please. 

Otherwise all good! Thanks again!

Reply:
done